dispatch/static/manager/src/js/index.js

```diff
@@ -100,6 +100,7 @@ render((
           <Route path='events'>
             <IndexRoute component={Pages.Events.Index} />
             <Route path='new' component={Pages.Events.NewEvent} />
+            <Route path='audit' component={Pages.EventAudit} /> 
```

Comment:
The audit page should be grouped with the other events pages and referenced like so:

`component={Pages.Events.Audit}`